server/app/graphql/resolvers/drugs.rb

```diff
@@ -37,6 +37,10 @@ class Resolvers::Drugs < GraphQL::Schema::Resolver
     scope.where(concept_id: value)
   end
 
+  option(:concept_ids, type: String, description: 'Exact match filtering on concept IDs') do |scope, value|
+    scope.where('concept_id IN (?)', value)
```

Comment:
I'm a little confused at this one, `IN` is generally used for a list, and this argument is typed as a single string. Is it intended to be `[String]`. Additionally, doesn't matter a ton, but ActiveRecord is smart enough to handle an array with an `IN` for you, so if you change it, you can just pass in the array directly with `scope.where(concept_id: value)`.

Reply:
Good catch, this is definitely a goof